app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanel.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect, useState } from 'react';
+import { cx } from '@emotion/css';
+import { ConnectionPanelMain } from './ConnectionPanelMain';
+import { ConnectionPanelTop } from './ConnectionPanelTop';
+import { connectionPanelContainer } from './ConnectionPanel.styles';
+import { ConnectedState } from '../StatusLabel/StatusLabel';
+
+const ConnectionPanel = (): JSX.Element => {
+	// TODO - remove temp state and define pending/duplicate/connected state from data
```

Comment:
It'd be great to connect these TODOs to tickets so that we can make sure nothing gets lost/forgotten

Reply:
I'm currently adding them to a trello board and will convert them to tickets when I start working on the functionality/pulling data in. Trying to focus on the UI stuffs first.